dist/images/clean-ic-az-db.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
We should describe when to use this script and how to get the `avaliability_zone_uuid` in the script

Reply:
it's described in the doc on our own cf.  
https://confluence.alauda.cn/pages/viewpage.action?pageId=168312049
and it may need the user to find the garbage available_zone itself

Reply:
![Uploading image.png…]()

Reply:
fixed